rabbitinahat/src/main/java/org/ohdsi/rabbitInAHat/RabbitInAHatMain.java

```diff
@@ -330,6 +331,10 @@ private JMenuBar createMenuBar() {
 		addStemTable.addActionListener(this);
 		editMenu.add(addStemTable);
 
+		JMenuItem removeStemTable = new JMenuItem(ACTION_REMOVE_STEM_TABLE);
+		removeStemTable.addActionListener(this);

```

Comment:
FYI: you can use a lambda here to directly reference the method you want to run. It’s not necessary for this PR but would improve coherence.